packages/runtime-core/src/apiWatch.ts

```diff
@@ -285,7 +285,11 @@ function doWatch(
     // we will also not call the invalidate callback (+ runner is not set up)
     onCleanup = NOOP
     if (!cb) {
-      getter()
+      if (flush === 'post') {
```

Comment:
This's not the proper fix.
In SSR there is no need to setup an effect.

Reply:
Thanks, then i close it. Looking forward to your solution.

Reply:
`useCssVars` only work at runtime.
I believe we should not inject `useCssVar()` at the server.